packages/sol-compiler/package.json

```diff
@@ -74,11 +75,15 @@
         "@0x/typescript-typings": "^3.0.8",
         "@0x/utils": "^3.0.1",
         "@0x/web3-wrapper": "^3.2.4",
+        "@resolver-engine/core": "^0.2.1",
```

Comment:
`@resolver-engine/imports-fs` exports all functionality of core and imports, you can just add that one dependency and import everything from there